tchannel/thrift/client.py

```diff
@@ -166,17 +167,34 @@ def send(self, *args, **kwargs):
 
         body = serializer.serialize_body(call_args)
         header = serializer.serialize_header({})
-        response = yield self.tchannel.request(
-            hostport=self.hostport, service=self.service
-        ).send(
-            arg1=endpoint,
-            arg2=header,
-            arg3=body,  # body
-            headers=self.protocol_headers,
-            traceflag=self.trace
-        )
-        body = yield response.get_body()
+
+        # Glue for old API.
+        if hasattr(self.tchannel, 'request'):
+            response = yield self.tchannel.request(
+                hostport=self.hostport, service=self.service
+            ).send(
+                arg1=endpoint,
+                arg2=header,
+                arg3=body,  # body
+                headers=self.protocol_headers,
+                traceflag=self.trace
+            )
+            body = yield response.get_body()
+        else:
+            response = yield self.tchannel.call(
+                scheme=schemes.THRIFT,
+                service=self.service,
+                arg1=endpoint,
+                arg2=header,
+                arg3=body,
+                hostport=self.hostport,
+                #headers=self.protocol_headers,
+                #traceflag=self.trace,
+            )
```

Comment:
Some people are still using `tchannel.tornado.TChannel` which doesn't have the new API though.